include/votca/xtp/cudapipeline.h

```diff
@@ -0,0 +1,71 @@
+/*
+ *            Copyright 2009-2019 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef __XTP_CUDA_PIPELINE__H
+#define __XTP_CUDA_PIPELINE__H
+
+#include <cublas_v2.h>
```

Comment:
I would add a:
```
#include <votca_config.h>
#ifndef USE_CUDA
#error Cuda not enabled
#endif
```
To prevent developers from accidentally including it.

Reply:
Thanks for the hint!

Reply:
I have added it